### DISCUSSION TITLE
Idea: Shortcut Click Action by Category

### DISCUSSION THREAD

**Discussion by JNavas2** [Waboodoo/HTTP-Shortcuts#271] [Ideas]
This is so minor that I'm almost embarrassed to suggest it, but I think it would be cool to be able to override the global Shortcut Click Action in a specific Category.

In most Categories I would like the Shortcut Click Action to be Edit, but I have a few categories where I want it to be Execute. My use case for these categories is in a widget on my Home screen. For example, my category Music has Browser Shortcuts to my individual YouTube Music playlists, so that in just two quick taps I can open a desired playlist as long as I have the global Shortcut Click Action set to Execute, but that means it can't be Edit in other Categories.

Again, this is minor, just food for thought.

**Comment by Waboodoo** [comment]
Thanks for the idea. I'm was planning on adding some more fine-grained options to how categories can be configured, so this might fit in with that. I'll keep it in mind.

**Reply by JNavas2** [reply]
Thank you!

**Reply by Waboodoo** [reply]
This feature is added in version 2.15.0, which will be released within the next few days.

**Reply by JNavas2** [reply]
> This feature is added in version 2.15.0, which will be released within the next few days.

Works great, thank you!
